Makefile

```diff
@@ -5,117 +5,59 @@
 # container images of the osbuild-containers repository.
 #
 
-#
-# Global Setup
-#
-# This section sets some global parameters that get rid of some old `make`
-# annoyences.
-#
-#     SHELL
-#         We standardize on `bash` for better inline scripting capabilities,
-#         and we always enable `pipefail`, to make sure individual failures
-#         in a pipeline will be treated as failure.
-#
-#     .SECONDARY:
-#         An empty SECONDARY target signals gnu-make to keep every intermediate
-#         files around, even on failure. We want intermediates to stay around
-#         so we get better caching behavior.
-#
-
-SHELL			:= /bin/bash -eox pipefail
-
-.SECONDARY:
-
-#
-# Parameters
-#
-# The set of global parameters that can be controlled by the caller and the
-# calling environment.
-#
-#     BUILDDIR
-#         Path to the directory used to store build artifacts. This defaults
-#         to `./build`, so all artifacts are stored in a subdirectory that can
-#         be easily cleaned.
-#
-#     SRCDIR
-#         Path to the source code directory. This defaults to `.`, so it
-#         expects `make` to be called from within the source directory.
-#
-#     BIN_*
-#         For all binaries that are executed as part of this makefile, a
-#         variable called `BIN_<exe>` defines the path or name of the
-#         executable. By default, they are set to the name of the binary.
-#
-
-BUILDDIR		?= ./build
-SRCDIR			?= .
+PACKAGE_NAME = osbuild-containers
```

Comment:
This variable does not seem to be used anywhere.

Reply:
Fixed